frontend/src/components/Chat/ChatMessage.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-list-item class="messageItem">
+    <v-list-item-avatar>
+      <chat-message-menu
+        :targetId="senderId"
+        @chat-message-menu-selection="
+          $emit('chat-message-menu-selection', $event)
+        "
+      >
+        <v-img
+          max-height="45"
+          max-width="45"
+          :src="'https://static.generated.photos/vue-static/home/hero/3.png'"
```

Comment:
For now the store getAvatar() can only fetch the current user right ? Should we make it so that it can fetch any avatar, and should we store other users avatar as well ? It is still a bit confusing to me, but it seems like we might do a lot of requests if we don't.

Reply:
Yeah, I think that would be a good idea

I had opened PR #148  to address the first issue. I think it would be a good idea to store them as well, particularly for the chat if we show the avatar on every message, so maybe we can open an issue for that

Reply:
I opened #161 to address this question